Cargo.toml

```diff
@@ -29,7 +30,7 @@ unsigned-varint = { version = "0.7.1", features = ["futures", "std"] }
 [dev-dependencies]
 async-std = { version = "1.10.0", features = ["attributes"] }
 env_logger = "0.9.0"
-libipld = { version = "0.15.0", default-features = false, features = ["dag-cbor"] }
-libp2p = { version = "0.50.0", features = ["tcp", "noise", "yamux", "rsa", "async-std"] }
-multihash = { version = "0.17.0", default-features = false, features = ["blake3", "sha2"] }
+libipld = { version = "0.16.0", default-features = false, features = ["dag-cbor"] }
+libp2p = { version = "0.51.3", features = ["tcp", "noise", "yamux", "rsa", "async-std"] }
+multihash = { version = "0.18", default-features = false, features = ["blake3", "sha2"] }
```

Comment:
Note that rust-libp2p still uses multihash v0.17.0. See comment here https://github.com/ipfs-rust/libp2p-bitswap/pull/43/files#r1200305495.